volatility3/cli/__init__.py

```diff
@@ -360,10 +360,9 @@ def run(self):
         subparser = parser.add_subparsers(
             title="Plugins",
             dest="plugin",
-            description="For plugin specific options, run '{} <plugin> --help'".format(
-                self.CLI_NAME
-            ),
+            description=f"For plugin specific options, run '{self.CLI_NAME} <plugin> --help'",
             action=volargparse.HelpfulSubparserAction,
+            metavar="PLUGIN",
```

Comment:
Absolutely fine, I didn't realize it would tidy things up so much, but very happy to have it clearer up!  5:)